build.gradle.kts

```diff
@@ -79,6 +79,13 @@ allprojects {
         }
     }
 
+
+    tasks.register("printClasspath") {
```

Comment:
A comment explaining the usage of this task would be helpful

Reply:
Updated.